lineman/spec/helpers/factory.js

```diff
@@ -0,0 +1,70 @@
+window.useFactory = function() {
```

Comment:
This is a helper class similar to FactoryGirl for easily making models in the clientside db.